src/server/node_manager.c

```diff
@@ -3033,7 +3036,12 @@ deallocate_job(mominfo_t *pmom, job *pjob)
 		free(new_exec_vnode);
 
 	}
-	set_scheduler_flag(SCH_SCHEDULE_TERM);
+	if (find_assoc_sched_jid(pjob->ji_qs.ji_jobid, &psched))
+		set_scheduler_flag(SCH_SCHEDULE_TERM, psched);
+	else {
+		sprintf(log_buffer, "Unable to find scheduler associated with partition");
+		log_err(-1, __func__, log_buffer);
```

Comment:
I guess you can just give the string here. copying message into buffer isn't helping

Reply:
Done